### DISCUSSION TITLE
Bortle Index modification with Web API ?

### DISCUSSION THREAD

**Discussion by WilliamDrap** [Stellarium/stellarium#2613] [Q&A] [answered]
Hello,
I want to make my own remote control interface.  Is it possible to modify the BorteScale Index with the Web API and How ? 
Thanks for advance
William

**Comment by gzotti** [comment, marked as answer]
This is something to be further clarified and modified in the next weeks. Maybe only in the December release.

**Reply by WilliamDrap** [reply]
Ok. Thank You
